docs/ccoctl.md

```diff
@@ -161,3 +161,55 @@ $ ccoctl gcp delete --name=<name> --project=<gcp-project-id>
 ```
 
 where `name` is the name prefix used to create cloud resources. `project` is the ID of the gcp project.
+
+## IBMCloud
+
+### Global flags
+
+By default, the tool will output to the directory the command(s) were run in. To specify a directory, use the `--output-dir` flag.
+
+### Extract the Credentials Request objects from the above release image
+
+`ccoctl ibmcloud` can process two kind of credentials requests - `IBMCloudProviderSpec`, `IBMCloudPowerVSProviderSpec` and here are the steps to extract them from the release image
+
+#### IBM Cloud
+
+This extracts the credentials of kind `IBMCloudProviderSpec`
+
+```bash
+mkdir credreqs ; oc adm release extract --cloud=ibmcloud --credentials-requests $RELEASE_IMAGE --to=./credreqs
+```
+
+#### IBM Cloud Power VS
+
+This extracts the credentials of kind `IBMCloudPowerVSProviderSpec`
+
+```bash
+mkdir credreqs ; oc adm release extract --cloud=powervs --credentials-requests $RELEASE_IMAGE --to=./credreqs
+```
+
+### Creating Service IDs
+
+This command will create the service ID for each credential request, assign the policies defined, creates an API key in the IBM Cloud and generates the secret.
+
+```bash
+ccoctl ibmcloud create-service-id --credentials-requests-dir <path-to-directory-with-list-of-credentials-requests> --name <name> --resource-group-name <resource-group-name>
+```
+
+> Note: --resource-group-name option is optional, but it is recommended to use to have finer grained access to the resources. 
+
+### Refresh the API keys for Service ID
+
+```bash
+ccoctl ibmcloud refresh-keys --kubeconfig <openshift-kubeconfig-file> --credentials-requests-dir <path-to-directory-with-list-of-credentials-requests> --name <name> 
+```
```

Comment:
@joelddiaz  If there are new added credentialsrequests in credentials-requests-dir,  need `--create` parameter , otherwise will hit some error like the below, not sure if this one worth to add to doc.
<pre>
Error: Failed to refresh keys: ServiceID: lwan-ibm-svcid-openshift-machine-api-ibmcloud-cloud-credentials does not exist, rerun with --create flag to create it
</pre>

Reply:
Thanks @lwan-wanglin . @mkumatag it's probably worth a note that the example command will only refresh existing creds, but any new creds int he cred-req-dir will require the --create parameter

Reply:
added the commit for that.